llvm/lib/Target/AIE/AIEMachineScheduler.cpp

```diff
@@ -616,34 +632,15 @@ void AIEPostRASchedStrategy::leaveRegion(const SUnit &ExitSU) {
   assert(BS.getCurrentRegion().Bundles.empty());
   BS.addBundles(TopBundles);
   BS.addBundles(BotBundles);
+  if (!ReAssignMultiSlotInstr)
+    materializeMultiSlotInstrs();
```

Comment:
Why dot keep the materialization call where it used to be? Is there any reason why you now do it after computing the Bundles?